internal/dockercompose/fake_client.go

```diff
@@ -80,8 +81,8 @@ func (c *FakeDCClient) StreamEvents(ctx context.Context, pathToConfig string) (<
 				select {
 				case events <- event: // send event to channel (unless it's full)
 				default:
-					c.t.Fatalf("no room on events channel to send event: '%s'. Something "+
-						"is wrong (or you need to increase the buffer).", event)
+					panic(fmt.Sprintf("no room on events channel to send event: '%s'. Something "+
```

Comment:
Does this fail the test still if this condition is hit?

Reply:
yeah, but it's not as pretty, e.g.:
```
[11:45:00 matt@mbp tilt (master)]$ go test ./internal/engine -run '^TestFoo$'
panic: hi

goroutine 9 [running]:
github.com/windmilleng/tilt/internal/engine.TestFoo.func1()
	/Users/matt/go/src/github.com/windmilleng/tilt/internal/engine/buildcontroller_test.go:144 +0x39
created by github.com/windmilleng/tilt/internal/engine.TestFoo
	/Users/matt/go/src/github.com/windmilleng/tilt/internal/engine/buildcontroller_test.go:143 +0x35
FAIL	github.com/windmilleng/tilt/internal/engine	0.057s
[12:00:26 matt@mbp tilt (master *)]$
```

Reply:
but at least it fails instead of no-opping!